features/support/helpers/capybara_helpers.rb

```diff
@@ -22,7 +22,8 @@ def assert_path_returns_error(path, status_code: 403)
     end
     # HACK: the error page requests assets that return 200 and checking the first one is not always
     # right. As an easy workaround, rule out requests with an url to assets.
-    requests.reject { |request| request.url&.include?('/assets/') }.first.status_code.should == status_code
+    request = requests.reject { |request| request.url&.include?('/assets/') }.first
+    assert_equal status_code, request.status_code, "Expected: #{status_code} Actual: #{request.status_code} for #{request.url}"
```

Comment:
this is not an actual fix. Rather if something goes wrong we will see which ulr it failed with to hopefully make this hack more reliable. This is an effort to in future know more about failures like

https://app.circleci.com/pipelines/github/3scale/porta/27372/workflows/d0505ef3-699f-49d0-8923-ddbcd1f92210/jobs/306904/tests